internal/cli/orgs.go

```diff
@@ -66,3 +77,57 @@ var CmdOrgCreate = &cobra.Command{
 	SilenceErrors: true,
 	SilenceUsage:  true,
 }
+
+// CmdOrgDelete implements the epinio `orgs delete` command
+var CmdOrgDelete = &cobra.Command{
+	Use:   "delete NAME",
+	Short: "Deletes an organization",
+	Args:  cobra.ExactArgs(1),
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		force, err := cmd.Flags().GetBool("force")
+		if err != nil {
+			return err
+		}
+		if !force {
+			cmd.Printf("You are about to delete organization %s and everything included in it (applications, services etc). Are you sure? (Y/n): ", args[0])
```

Comment:
When the user simply hits `Enter` the see the message:

```
You are about to delete organization workspace and everything included in it (applications, services etc). Are you sure? (Y/n): 
Please enter Y or n: 
```
That means `Y` is not the default option thus it shouldn't be capitalized. Should be `y/n` instead.

Reply:
also in the verification message (^ above), it should also be in lower case.

Reply:
Done. Changed it to lowercase y.